sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2415,8 +2415,7 @@ object GpuOverrides {
       (c, conf, p, r) => new ExprMeta[CollectList](c, conf, p, r) {
         override def convertToGpu(): GpuExpression = GpuCollectList(
           childExprs.head.convertToGpu(), c.mutableAggBufferOffset, c.inputAggBufferOffset)
-      }).disabledByDefault("for now the GPU collects null values to a list, but Spark does not." +
-      " This will be fixed in future releases."),
+      }).disabledByDefault("for now it is only supported by windowing."),
```

Comment:
Do we have a test that shows that we try to run this in the case of a groupby or aggregation? `ExprChecks.aggNotGroupByOrReduction` should take care of only letting it happen as a window function.  If that is not happening, then we need to fix that code instead of turning off the operator

Reply:
Sorry for confusion.
Actually it is still only supported by windowing. This change here focuses on the doc update about the fix of excluding null values.
I changed to enable it by default. As you said, we have `aggNotGroupByOrReduction ` to make sure the context. In windowing, `collect_list ` works as what Spark version does now.